Sources/Swift/Integrations/UserFeedback/Configuration/SentryUserFeedbackThemeConfiguration.swift

```diff
@@ -88,24 +88,18 @@ public class SentryUserFeedbackThemeConfiguration: NSObject {
      */
     public var buttonBackground: UIColor = UIColor.clear
     
-    /**
-     * Color used for success-related components (such as text color when feedback is submitted successfully).
-     * - note: Default light mode: `rgb(38, 141, 117)`; dark mode: `rgb(45, 169, 140)`
-     */
-    public var successColor = UIScreen.main.traitCollection.userInterfaceStyle == .dark ? UIColor(red: 45 / 255, green: 169 / 255, blue: 140 / 255, alpha: 1) : UIColor(red: 38 / 255, green: 141 / 255, blue: 117 / 255, alpha: 1)
-    
     /**
      * Color used for error-related components (such as text color when there's an error submitting feedback).
      * - note: Default light mode: `rgb(223, 51, 56)`; dark mode: `rgb(245, 84, 89)`
      */
     public var errorColor = UIScreen.main.traitCollection.userInterfaceStyle == .dark ? UIColor(red: 245 / 255, green: 84 / 255, blue: 89 / 255, alpha: 1) : UIColor(red: 223 / 255, green: 51 / 255, blue: 56 / 255, alpha: 1)
     
-    public struct OutlineStyle: Equatable {
+    @objc public class SentryFormElementOutlineStyle: NSObject {
         /**
          * Outline color for form inputs.
          * - note: Default: The system default of a UITextField outline with borderStyle of .roundedRect.
          */
-        public var outlineColor = UIColor(white: 204 / 255, alpha: 1)
+        public var color = UIColor(white: 204 / 255, alpha: 1)
```

Comment:
since this is already scoped under something called "outline style", the "outline" on this property was redundant